app/routes.js

```diff
@@ -0,0 +1,51 @@
+var ensureLoggedIn = require('connect-ensure-login').ensureLoggedIn;
+var express = require('express');
+var middleware = require('./middleware');
+var passport = require('passport');
+var proxy = require('./proxy');
+var router = new express.Router();
+
+
+router.get('/login', function(req, res) {
+  res.render('login.html', {env: process.env});
```

Comment:
Shouldn't the views now use the config rather the environment variables directly?

Reply:
Oops! yes they should